detection_rules/main.py

```diff
@@ -94,8 +94,9 @@ def generate_rules_index(ctx: click.Context, query, overwrite, save_files=True):
 
 @root.command('import-rules')
 @click.argument('input-file', type=click.Path(dir_okay=False, exists=True), nargs=-1, required=False)
+@click.option('--required-only', is_flag=True, help='Only prompt for required fields')
```

Comment:
Should we add a default?

Reply:
Agreed. currently for me in testing, I think it defaults to `False` 

<details><summary>Test Run</summary>
<p>

```shell
detection-rules on  update_import_rules [$!?] is  v0.1.0 via  v3.12.2 (detection-rules-build) on  eric.forte
❯ python -m detection_rules import-rules /home/forteea1/Code/clean_mains/detection-rules/exports/20240314T190337L.ndjson

█▀▀▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄▄▄ ▄   ▄      █▀▀▄ ▄  ▄ ▄   ▄▄▄ ▄▄▄
█  █ █▄▄  █  █▄▄ █    █   █  █ █ █▀▄ █      █▄▄▀ █  █ █   █▄▄ █▄▄
█▄▄▀ █▄▄  █  █▄▄ █▄▄  █  ▄█▄ █▄█ █ ▀▄█      █ ▀▄ █▄▄█ █▄▄ █▄▄ ▄▄█

[+] Building rule for /home/forteea1/Code/clean_mains/detection-rules/rules/threat_intel_ip_address_indicator_match.toml
actions (multi, comma separated): test
alert_suppression:
building_block_type:
concurrent_searches:
data_view_id:
enabled:
exceptions_list (multi, comma separated):
false_positives (multi, comma separated):
filters (multi, comma separated):
items_per_search:
max_signals:
meta:
related_integrations (multi, comma separated):
risk_score_mapping (multi, comma separated):
rule_name_override:
severity_mapping (multi, comma separated):
add mitre tactic? [y/N]:
throttle:
to:


```

</p>
</details>

Reply:
I was thinking this but it is a flag. We could rename it to something that's like `import-all-fields`. I also went with this field name to be consistent with the call to `create-rule`. Thoughts?